RELEASING.rst

```diff
@@ -39,23 +56,15 @@ Updating the online documentation site
     This instructions are currently out-of-date and are to be considered under
     construction.
 
-* Go to the doc directory::
-
-  $ cd doc
-
-* Make sure that the ``version``/``release`` variables are updated in
-  ``conf.py``.
+* Build the html::
 
-* Make the html version of the docs::
-
-  $ rm -rf _build/html
-  $ make html
+  $ python setup.py build_sphinx
 
 * Make a backup and upload the files in the doc site (xodo)::
 
   $ export UPSTREAM="/home/blosc/srv/www/bcolz.blosc.org"
   $ ssh blosc@xodo.blosc.org "mv $UPSTREAM/docs/html $UPSTREAM/docs/html.bck"
-  $ scp -r _build/html blosc@xodo.blosc.org:$UPSTREAM/docs
+  $ scp -r build/sphinx/html blosc@xodo.blosc.org:$UPSTREAM/docs
```

Comment:
yeah, this is actually all not true anymore. I actually do a very different and custom dance to deploy the docs due to the way in which I use xodo and the way the docs are deployed. I'll fix the instructions at some later point, so maybe you could just say: "WARNING: these instructions are out of date" here.